### PR TITLE
jenkins: new, 2.468

### PR DESCRIPTION
**Pull request by rickliu2000**
Topic Description
-----------------

- jenkins: new, 2.468

Package(s) Affected
-------------------

- jenkins: 2.468

Security Update?
----------------

No

Build Order
-----------

```
#buildit jenkins
```

Test Build(s) Done
------------------

### CODE REVIEW

**Review by MingcongBai — CHANGES_REQUESTED** [review]
图片

From @rickliu2000

**Comment by rickliu2000** [comment]
> 图片
> 
> From @rickliu2000

image

This is caused by the proxy. Using a transparent proxy instead eliminates the issue.

File: app-admin/jenkins/autobuild/build [hunk 1]
```diff
@@ -0,0 +1,6 @@
+abinfo "Building Jenkins..."
+mvn -am -pl war,bom -Pquick-build -Dproject.build.outputTimestamp=`date +%s` package
+
+abinfo "Installing Jenkins..."
+install -Dm 644 war/target/jenkins.war "$PKGDIR"/usr/share/java/jenkins/jenkins.war
```
Comment (MingcongBai) [inline app-admin/jenkins/autobuild/build]: Use `-Dvm644` , also use absolute paths:

```
"$SRCDIR"/war/target/jenkins.war
```

File: app-admin/jenkins/autobuild/postinst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+systemd-sysusers jenkins.conf
+systemd-tmpfiles --create jenkins.conf
+[[ -d var/lib/jenkins ]] || install -dm 750 -o jenkins -g jenkins var/lib/jenkins
```
Comment (MingcongBai) [inline app-admin/jenkins/autobuild/postinst]: This should be handled by tmpfiles.d.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-admin/jenkins/autobuild/postinst (hunk 2 above)
Comment (rickliu2000) [reply app-admin/jenkins/autobuild/postinst:3]: In reality, it didn't.
I have to manually run the two systemd commands to create the user/group and tmpfile folder.

File: app-admin/jenkins/autobuild/build (hunk 1 above)
Comment (rickliu2000) [reply app-admin/jenkins/autobuild/build]: changed

File: app-admin/jenkins/autobuild/postinst (hunk 2 above)
Comment (rickliu2000) [reply app-admin/jenkins/autobuild/postinst]: changed
now handled by tmpfiles.d

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**